components/Col/index.js

```diff
@@ -0,0 +1,14 @@
+import PropTypes from 'prop-types';
+import {View} from 'react-native';
+import styles from './style';
+
+const Col = ({numRows, children}) => {
+  return <View style={styles[`${numRows}col`]}>{children}</View>;
```

Comment:
add screenshot why did you do it?

Reply:
To arrange pictures in a mixed order, similar to a figma arrangement. 

![photo_2023-06-27_12-20-18](https://github.com/MiqayelChaloyan/demo_native_app/assets/108179437/e5bfdf91-477e-4e1d-b10c-a30506421478)
![Screenshot from 2023-06-27 12-16-51](https://github.com/MiqayelChaloyan/demo_native_app/assets/108179437/7839b5f4-0e9e-40fb-9186-7dcd7da22890)